### PR TITLE
Add documentation for --icons flag

### PR DESCRIPTION
**Pull request by filalex77**
- [X] Update completions
  - [X] fish
  - [X] zsh
- [X] Update man page
- [X] Update help text

9497b30b3322c52e9c8117a97abd6d96001dac65

### CODE REVIEW

**Comment by ogham** [comment]
Thank you! 👍

File: contrib/completions.fish [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```
Comment (filalex77) [inline contrib/completions.fish]: Whoops, I think this one was added automatically...

Comment (ogham) [reply contrib/completions.fish]: heh, it was at the very top of the file so I didn't notice. I'll fix it

Comment (filalex77) [reply contrib/completions.fish]: Thanks!

**Pull request merged**